docs/src/main/sphinx/release/release-409.md

```diff
@@ -20,6 +27,12 @@
 * Remove support for specifying expressions in the `sample_by` table property to
   prevent SQL injection. ({issue}`16261`)
 
+## Delta Lake connector
+
+* Automatically infer required Delta Lake version when creating new tables or
+  enabling new table features. ({issue}`16310`)
```

Comment:
what is "enabling new table features" ?

Reply:
This doesn't sound like it has any user-visible impact, does it? Or would it be something like "Avoid failures when ..."